packages/flutter/lib/src/widgets/text_editing_intents.dart

```diff
@@ -234,6 +234,23 @@ class ScrollToDocumentBoundaryIntent extends DirectionalTextEditingIntent {
   }) : super(forward);
 }
 
+/// Scrolls up or down by page depending on the [forward] parameter.
+class ScrollByPageIntent extends DirectionalTextEditingIntent {
+  /// Creates a [ScrollByPageIntent].
+  const ScrollByPageIntent({
+    required bool forward,
+  }) : super(forward);
```

Comment:
I thought we had the super parameter lint enabled?

Reply:
This confused me for a second.  It looks like the `forward` param is positional in DirectionalTextEditingIntent ([code](https://github.com/flutter/flutter/blob/a494a12bb0c42b05d110dc7d3082ca6585f1c6ff/packages/flutter/lib/src/widgets/text_editing_intents.dart#L23-L25)).  Is that intentional?  I guess I will make this one positional too, but tell me if you think there is a better approach.  Making this positional is a departure from the other Intens in DefaultTextEditingIntents that all have a `forward` named parameter.

Reply:
Actually this is obsolete if we replace this with ScrollIntent as in https://github.com/flutter/flutter/pull/105497/commits/d4bbc340023000765222983b17ec53709024cf37.